src/hooks/useChats.ts

```diff
@@ -0,0 +1,23 @@
+import { useFirestoreConnect } from "react-redux-firebase";
```

Comment:
We should try and keep the filenames and the functions they export aligned. So `useChats` doesn't make sense for this filename (as it makes me think that there should be a `useChats` function being exported, which there isn't)

I would either make the filename `useVenueChats` or similar, or make it more generic, and just call it `chat` or similar.

Reply:
Also, instead of importing `useFirestoreConnect` directly from the lib, I believe we have our own typed wrappers we should be using instead (assuming that PR has been merged?)

Reply:
Fixed in 354167a550c42e93a27cc5c1e431145ff1caeef7